src/hotspot/share/prims/jvmtiEnv.cpp

```diff
@@ -450,6 +450,18 @@ JvmtiEnv::RetransformClasses(jint class_count, const jclass* classes) {
 
     InstanceKlass* ik = InstanceKlass::cast(klass);
     if (ik->get_cached_class_file_bytes() == nullptr) {
+      // Link the class to avoid races with the rewriter. This will call the verifier also
+      // on the class. Linking is done already below in VM_RedefineClasses below, but we need
```

Comment:
```suggestion
      // on the class. Linking is also done in VM_RedefineClasses below, but we need
```
There are two "below"s